tests/evaluation/independent_body2/independent_body2.err

```diff
@@ -0,0 +1,6 @@
+Warning: Variable y only occurs once in file independent_body2.dl at line 55
```

Comment:
y occurs twice in the line, let alone file. Could the error be clarified a little?

Reply:
This is an artifact of the expansion. With disjunctions in the representation, we can take care of this.